File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.cs
using System;

public class Program {
    public static void Main() {
        string[] split = Console.ReadLine().Split(new char[] { ' ' }, StringSplitOptions.None);
        int l = Int32.Parse(split[0]);
        int r = Int32.Parse(split[1]);
        int val = 2*(l > r ? l : r);
        if (val == 0) { Console.WriteLine("Not a moose"); }
        else if (l == r) { Console.WriteLine("Even " + val); }
        else { Console.WriteLine("Odd " + val); }
    }
}

File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.pas
program different;

uses sysutils, math;

var
    l : longint;
    r : longint;

begin
   read(l);
   read(r);
   if ((l = 0) and (r = 0)) then
      writeln('Not a moose')
   else if (l = r) then
      writeln('Even ' + IntToStr(l+r))
   else
      writeln('Odd ' + IntToStr(2*Max(l, r)));
end.
